rslib/src/scheduler/answering/mod.rs

```diff
@@ -59,7 +61,7 @@ impl CardStateUpdater {
     /// state handling code from the rest of the Anki codebase.
     pub(crate) fn state_context(&self) -> StateContext<'_> {
         StateContext {
-            fuzz_seed: self.fuzz_seed,
+            fuzz_factor: get_fuzz_factor(self.fuzz_seed),
```

Comment:
I (ab?)used this to work around the different algorithm in `with_learning_fuzz()`.